scripts/primaza.sh

```diff
@@ -25,9 +25,6 @@ GIT_SHA_COMMIT=$(git rev-parse --short HEAD)
 PRIMAZA_GITHUB_REPO=https://github.com/halkyonio/primaza-poc
 HALKYONIO_HELM_REPO=https://halkyonio.github.io/helm-charts/
 PRIMAZA_IMAGE_NAME=${PRIMAZA_IMAGE_NAME:-quay.io/halkyonio/primaza-app:${GIT_SHA_COMMIT}}
-VAULT_URL=${VAULT_URL:-vault.127.0.0.1.nip.io}
-VAULT_USER=${VAULT_USER:-bob}
-VAULT_PWD=${VAULT_PWD:-sinclair}
```

Comment:
Why are you removing such variables as we need them to define default values when we use vault locally (kind, etc) ?
They can be override if you set them externally:
```
export VAULT_URL=vault.<VM_IP.nip.io
./script/primaza.sh
```

Reply:
> Why are you removing such variables as we need them to define default values when we use vault locally (kind, etc) ? They can be override if you set them externally:
> 
> ```
> export VAULT_URL=vault.<VM_IP.nip.io
> ./script/primaza.sh
> ```
Default values are already configured in the application-kubernetes.properties file. We could use here the env  var to override these values but to set the same

Reply:
I agree for the values `vault.user` and `vault.password` but this is not true for the `vault.url` as vault can run separately from primaza-app, can be deployed on another k8s cluster, can run as standalone application (or even as docker container, etc and by consequence it is needed that we can override, when we use the bash script, the `vault.url`. 

We should nevertheless offer the possibility, part of the bash script which is also responsible to declare the Helm env var of the primaza app to set the 3 different vault values as they could change according to where vault is running, how the user will set the user and password too

**Remark**: The `localDeploy` bash function supports to install primaza-app locally (= 127.0.0.1) be connected with the vault app running also locally (= vault.127.0.0.1.nip.io) and using the default values part of the application.properties. Such a function should be reviewed in a separate PR to merge the `localDeploy` and `deploy` functions as now we use Helm to install primaza and configure it.